bookwyrm/views/rss_feed.py

```diff
@@ -177,3 +178,66 @@ def item_link(self, item):
     def item_pubdate(self, item):
         """publication date of the item"""
         return item.published_date
+
+
+class RssShelfFeed(Feed):
+    """serialize a shelf activity in rss"""
+
+    description_template = "rss/edition.html"
+
+    def item_title(self, item):
+        """render the item title"""
+        if hasattr(item, "pure_name") and item.pure_name:
+            return item.pure_name
+        authors = item.authors
+        authors.display_name = f"{item.author_text}:"
+        template = get_template("rss/title.html")
+        return template.render({"user": authors, "item_title": item.title}).strip()
+
+    def get_object(
+        self, request, shelf_identifier, username
+    ):  # pylint: disable=arguments-differ
+        """the user who's posts get serialized"""
+        user = get_user_from_username(request.user, username)
+        # always get privacy, don't support rss over anything private
+        # Shelf.privacy_filter(request.user).filter(user=user).all()
+
+        # TODO: get the SHELF of the object
+        shelf = get_object_or_404(user.shelf_set, identifier=shelf_identifier)
+        shelf.raise_visible_to_user(request.user)
+        return shelf
+
+    def link(self, obj):
+        """link to the shelf"""
+        return obj.local_path
+
+    def title(self, obj):
+        """title of the rss feed entry"""
+        return _(f"Books added to {obj.name}")
+
+    def items(self, obj):
+        """the user's activity feed"""
+        return obj.books.order_by("-published_date")[:10]
```

Comment:
Should sort by shelved date, right?:

```suggestion
        return obj.books.order_by("-shelfbook__shelved_date")[:10]
```